src/stories/SForms.stories.tsx

```diff
@@ -60,6 +60,7 @@ const Template: ComponentStory<typeof SForms> = (
     horizontalWizardNav: horizontalNavBar,
     wizardStepButtons: true,
     ...getP("startingQuestionId", "layout-options-65"),
+    ...getP("startingQuestionOrigin", "hidden-question-1834"),
```

Comment:
does it make sense to have both startingQuestionId and startingQuestionOrigin defined at same time?